www/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "compe-plus-www",
+    "description": "Front end for CompE+",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^5.12.0",
+        "@testing-library/react": "^11.2.7",
+        "@testing-library/user-event": "^12.8.3",
+        "@types/jest": "^26.0.23",
+        "@types/node": "^12.20.13",
+        "@types/react": "^17.0.6",
+        "@types/react-dom": "^17.0.5",
+        "@types/react-redux": "^7.1.16",
+        "@types/redux": "^3.6.0",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-redux": "^7.2.4",
+        "react-scripts": "4.0.3",
+        "redux": "^4.1.0",
+        "typescript": "^4.2.4",
+        "web-vitals": "^1.1.2"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject",
+        "lint": "eslint src --ext .js,.ts,.tsx,.jsx",
+        "format": "prettier --config ../.prettierrc --ignore-path .prettierignore --write ."
+    },
+    "browserslist": {
+        "production": [
+            ">0.2%",
+            "not dead",
+            "not op_mini all"
+        ],
+        "development": [
+            "last 1 chrome version",
+            "last 1 firefox version",
+            "last 1 safari version"
+        ]
+    },
+    "devDependencies": {
```

Comment:
Shouldn't `typescript` be a dev dependency?

Reply:
I didn't know this but when using CRA it puts it in `dependencies`, according to [this](https://github.com/facebook/create-react-app/issues/6180) there isn't much distinction with dev dependencies or regular dependencies for React apps.

Reply:
Works for me